website/docs/docs/build/incremental-models.md

```diff
@@ -251,7 +251,7 @@ the reliability of your `unique_key`, or the availability of certain features.
 
 * [Snowflake](snowflake-configs#merge-behavior-incremental-models): `merge` (default), `delete+insert` (optional), `append` (optional)
 * [BigQuery](bigquery-configs#merge-behavior-incremental-models): `merge` (default), `insert_overwrite` (optional)
-* [Spark](spark-configs#incremental-models): `append` (default), `insert_overwrite` (optional), `merge` (optional, Delta-only)
+* [Databricks](databricks-configs#incremental-models): `append` (default), `insert_overwrite` (optional), `merge` (optional, Delta-only)
```

Comment:
```suggestion
* [Databricks](/reference/resource-configs/databricks-configs#incremental-models): `append` (default), `insert_overwrite` (optional), `merge` (optional, Delta-only)
```

Reply:
My concern about removing this is the spark adapter does support incremental models as well. Should we just create a new page here?